core/src/main/java/com/chopshop166/chopshoplib/sensors/PigeonGyro.java

```diff
@@ -46,27 +48,47 @@ public void close() throws Exception {
         // NoOp
     }
 
+
+    /**
+    * Sets the gyro's heading back to zero
+    */
     @Override
     public void reset() {
         gyro.setFusedHeading(0);
 
     }
 
+    
+    /** 
+     * @return The rate in which we have traveled in degrees per second
```

Comment:
Traveled or rotated?

Reply:
Also this function gives the current rate, so the description should be in present tense.